app/src/main/java/com/concordia/mcga/activities/MainActivity.java

```diff
@@ -16,10 +17,12 @@
 public class MainActivity extends AppCompatActivity {
 
     private DrawerLayout drawerLayout;
+    private static Context context;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        context = this;
```

Comment:
I didn't know about this method. I looked into it and it's the better one to chose in this context. 
Reference: http://stackoverflow.com/questions/22966601/what-is-different-between-mainactivity-this-vs-getapplicationcontext 

Will fix